.github/workflows/integration.yml

```diff
@@ -0,0 +1,36 @@
+name: Integration Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  job:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
+        node: ['10', '12', '14']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
```

Comment:
This is at v2, and also has built-in caching so you don't need to use `actions/cache` at all now :)

Reply:
yeah, not sure why this change show up in this pr - it's from the main branch afaik?